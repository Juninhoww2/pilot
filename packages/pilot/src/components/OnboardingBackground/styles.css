@import "former-kit-skin-pagarme/dist/styles/colors.css";
@import "former-kit-skin-pagarme/dist/styles/spacing.css";

.onboardingBackground {
  background-color: var(--color-squanchy-gray-20);
  display: flex;
  min-height: 100vh;
}

.logoAndImage {
  background: var(--color-white);
  color: var(--color-black);
  display: flex;
  flex-grow: 1;
  flex-direction: column;
  padding: 40px 64px;
}

.centerImage {
  align-self: center;
  margin: auto 0;
  width: 100%;
}

.children {
  box-sizing: border-box;
  padding: 40px 64px;
  width: 528px;
}

@media (min-width: 1440px) {

  .centerImage {
    max-width: 850px;
  }

  .children {
    padding-left: 112px;
    padding-right: 112px;
    width: 696px;
  }
}
