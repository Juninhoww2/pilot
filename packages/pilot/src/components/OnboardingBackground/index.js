import React from 'react'
import PropTypes from 'prop-types'
import styles from './styles.css'

import Logo from './logo.svg'
import WomanInBalance from '../../assets/woman-in-balance.svg'

const OnboardingBackground = ({ children }) => (
  <div className={styles.onboardingBackground}>
    <div className={styles.logoAndImage}>
      <Logo />
      <div className={styles.centerImage}>
        <WomanInBalance />
      </div>
    </div>

    <div className={styles.children}>
      {children}
    </div>
  </div>
)

OnboardingBackground.propTypes = {
  children: PropTypes.element.isRequired,
}

export default OnboardingBackground
