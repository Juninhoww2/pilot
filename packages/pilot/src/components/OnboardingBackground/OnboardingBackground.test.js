import React from 'react'
import { render } from '@testing-library/react'
import OnboardingBackground from '.'

test('OnboardingBackground should render', () => {
  const { container } = render(
    <OnboardingBackground>
      <p>Indiana Jones</p>
    </OnboardingBackground>
  )

  expect(container).toMatchSnapshot()
})
