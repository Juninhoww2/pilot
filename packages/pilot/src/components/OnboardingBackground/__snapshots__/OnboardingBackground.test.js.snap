// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OnboardingBackground should render 1`] = `
<div>
  <div
    class="onboardingBackground"
  >
    <div
      class="logoAndImage"
    >
      <svg
        data-file-name="SvgLogo"
      />
      <div
        class="centerImage"
      >
        <svg
          data-file-name="SvgWomanInBalance"
        />
      </div>
    </div>
    <div
      class="children"
    >
      <p>
        Indiana Jones
      </p>
    </div>
  </div>
</div>
`;
